src/python/tests/fasta/test_chunk.py

```diff
@@ -0,0 +1,441 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `ensembl.io.genomio.fasta.chunk` module.
+
+Typical usage example::
+    $ pytest test_chunk.py
+
+"""
+
+from contextlib import contextmanager, nullcontext
+from contextlib import nullcontext as does_not_raise
```

Comment:
```suggestion
from contextlib import contextmanager, nullcontext
```
If you need `nullcontext` for something else, you can leave out the alias (it may impact the readability of the code a little bit, but not too worrying).